src/main/java/cms/gov/madie/measure/services/MeasureService.java

```diff
@@ -316,13 +316,14 @@ public Page<MeasureListDTO> getMeasures(
 
   public void checkDuplicateCqlLibraryName(String cqlLibraryName) {
     if (StringUtils.isNotEmpty(cqlLibraryName)
-        && measureRepository.findByCqlLibraryName(cqlLibraryName).isPresent()) {
```

Comment:
Any idea why was this not working? is it because of multiple measures found?

Reply:
Yeah, it just dies if there's multiple found